src/components/rating/subcomponents/Star.jsx

```diff
@@ -3,19 +3,21 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import Icon, {TYPE, ICON_COLOR} from '../../icons/Icon';
 
-const Star = ({size, checked, onClick}) => {
+const Star = ({size, checked, onClick, index, ...props}) => {
   const starClass = classnames('sg-rate-box__star', {
     'sg-rate-box__star--checked': checked
   });
+  const onClickStar = () => onClick(index);
 
   return (
-    <span className={starClass} onClick={onClick}>
+    <span className={starClass} onClick={onClickStar} {...props}>
```

Comment:
`className` can be overwritten by `{...props}` - is it supposed to work like this?